fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AdminSetReplicaStatusStmtAnalyzer.java

```diff
@@ -0,0 +1,30 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.analyzer;
+
+import com.starrocks.analysis.AdminSetReplicaStatusStmt;
+import com.starrocks.common.AnalysisException;
+import com.starrocks.qe.ConnectContext;
+import com.starrocks.sql.ast.AstVisitor;
+
+public class AdminSetReplicaStatusStmtAnalyzer {
+    public static void analyze(AdminSetReplicaStatusStmt adminSetReplicaStatusStmt, ConnectContext session) {
+        new AdminSetStmtAnalyzerVisitor().visit(adminSetReplicaStatusStmt, session);
+    }
+
+    static class AdminSetStmtAnalyzerVisitor extends AstVisitor<Void, ConnectContext> {
+        public void analyze(AdminSetReplicaStatusStmt adminSetReplicaStatusStmt, ConnectContext session) {
+            visit(adminSetReplicaStatusStmt, session);
+        }
+
+        @Override
+        public Void visitAdminSetReplicaStatusStatement(AdminSetReplicaStatusStmt adminSetReplicaStatusStmt,
+                                                        ConnectContext session) {
+            try {
+                adminSetReplicaStatusStmt.checkProperties();
```

Comment:
The logic of checkProperties needs to be extracted, the new version does not depend on the code in the old version